plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSessionProperties.java

```diff
@@ -116,7 +126,7 @@ public List<PropertyMetadata<?>> getSessionProperties()
     @SuppressWarnings("unchecked")
     public static List<String> getColumnsToHide(ConnectorSession session)
     {
-        return (List<String>) session.getProperty(COLUMNS_TO_HIDE, List.class);
+        return (List<String>) session.getProperty(COLUMNS_TO_HIDE, List.class).stream().filter(column -> !INSTANT_TIME.equals(column)).collect(toUnmodifiableList());
```

Comment:
I don't think hard-coded `INSTANT_TIME` filter is a good idea. Please revert this change.